src/messages/messages.cc

```diff
@@ -0,0 +1,19 @@
+#include "messages.h"
+
+const char * node_message_id(int key) {
```

Comment:
Here, and throughout this PR, the `*` should lean to the left (`const char* node_message_id`).